### PR TITLE
eliminate-duplicate-datatype-instruction

### PR DESCRIPTION
**Pull request by eldrn1**
This is a suggestion for the request for help with eliminating overlapping and similar information about datatypes in this lesson. I'm a new Python learner myself, and I found the placement of the data type discussion here in between defining and reading variables a bit confusing. I suggest keeping discussion of datatypes very brief here, knowing that the 'data types and format' lesson will dive more in-depth into datatypes once learners have a better foundation of python. Hopefully these edits, or some form of them, is helpful!

### CODE REVIEW

**Comment by maxim-belkin** [comment]
Thanks, @eldrn1. Could you please clarify which part of the current material is/was confusing?

**Comment by eldrn1** [comment]
@maxim-belkin sorry for the delayed reply. I was referring to the part in the http://www.datacarpentry.org/python-ecology-lesson/01-short-introduction-to-Python/index.html lesson where it says "Everything in Python has a type. To get the type of something, we can pass it to the built-in function type:" and then gives an example. I feel this isn't the best placement in between 2 different ways to print the variable text= Data Carpentry. I think it'd flow better if went from ">>> text
"Data Carpentry"" step to ">>> print(text)
Data Carpentry" step directly, and save the part about the 'type' float etc. for a different part of the lesson. Does that make sense?

**Comment by maxim-belkin** [comment]
Makes sense. Thanks for the clarification. Can you rebase your PR on top of the latest gh-pages branch? Or do you want me to do that (I can do that because you allowed maintainers to add commits to your branch)?

**Comment by maxim-belkin** [comment]
Done!
And thank you for the interesting PR title: it reminds me of an anecdote:

>-Dearsupport.Thespacebarkeyonmykeyboarddoesnotwork.WhatshouldIdo?
-Hi!-real-programmers-don't-need-spacebar-key.

File: _episodes/01-short-introduction-to-Python.md [hunk 1]
```diff
@@ -43,16 +43,17 @@ Hello World
...
 One of the most basic things we can do in Python is assign values to variables:
 
 ~~~
-text = "Data Carpentry"  # An example of a string
-number = 42  # An example of an integer
-pi_value = 3.1415  # An example of a float
+text = "Data Carpentry"  # An example of assigning a value to a new text variable,
+                         # also known as a string data type in Python
+number = 42              # An example of assigning a numeric value, or an integer data type
+pi_value = 3.1415        # An example of assigning a value that is a float data type
```
Comment (maxim-belkin) [inline _episodes/01-short-introduction-to-Python.md]: I think this:
>...*value* that is a float *data* type

should be changed to:
> value of type `float`

or something like that

File: _episodes/01-short-introduction-to-Python.md [hunk 2]
```diff
@@ -43,16 +43,17 @@ Hello World
 {: .language-bash}
 
 
-## Introduction to Python built-in data types
+## Introduction to variables in Python
 
-### Strings, integers and floats
+### Assiging values to variables
```
[the remark this answers is not in the document]
Comment (tobyhodges) [reply _episodes/01-short-introduction-to-Python.md:48]:
```suggestion
### Assigning values to variables
```

**Comment by tobyhodges** [comment]
I'm helping the current lesson Maintainers process outstanding pull requests on this repository, in preparation for [transition to the new lesson infrastructure](https://carpentries.github.io/workbench/faq.html#update-fork-from-styles). Having made some minor adjustments, I think this PR is ready to be merged. Thank you for contributing, @eldrn1!

**Pull request merged**